src/redux/books/Books.js

```diff
@@ -0,0 +1,23 @@
+// Actions
+const ADD_BOOK = 'bookstore/booksReducer/ADD_BOOK';
+const REMOVE_BOOK = 'bookstore/booksReducer/REMOVE_BOOK';
+
```

Comment:
- [ ] Kindly rename this file from Book.js to book.js as in your configureStore.js book.js is mentioned which produces an error.

<img width="1728" alt="Screenshot 2022-06-22 at 7 23 49 AM" src="https://user-images.githubusercontent.com/70269507/174943090-604345f2-239d-4f80-aa1e-020631f3b4d0.png">

Reply:
- [ ] As suggested previously Kindly rename your file from Books.js to books.js as your app is having errors because of the wrong file name.
_You can fix this as shown below_

https://user-images.githubusercontent.com/70269507/174955665-4f465124-f6b7-4489-b2d8-c0ee7e38333b.mov